src/core/updateSelections/updateSelections.ts

```diff
@@ -66,7 +66,7 @@ export function getSelectionInfo(
  */
 export function selectionsToSelectionInfos(
   document: TextDocument,
-  selectionMatrix: Selection[][],
+  selectionMatrix: (readonly Selection[])[],
```

Comment:
Looks like our typescript version got bumped when I bumped yarn.lock, so now we need to be stricter about `readonly`